### DISCUSSION TITLE
Favoriten Backup

### DISCUSSION THREAD

**Discussion by anten1222** [osmandapp/OsmAnd#12222] [General]
🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑🛑

Please do not file questions or support requests on the GitHub issues tracker.

In case you aren't sure whether to open an issue or not, please send your request to support@osmand.net.

You can get your questions answered using the [Google group](https://groups.google.com/forum/#!forum/osmand) or chat in real-time via our Telegram instances: [English](https://t.me/OsmAndMaps), [German](https://t.me/deosmand), [French](https://t.me/frosmand), [Ukrainian](https://t.me/uaosmand), [Italian](https://t.me/itosmand), [Russian](https://t.me/ruosmand).

Osmand creates a backup for the favorites.  Now I wanted to restore the data.  Unfortunately it was not possible for me because I couldn't find a program on the smartphone or in the playstore that can decompose bz2

**Comment by scaidermern** [comment]
Maybe OsmAnd should offer to open these bz2 files directly.
Or favourite backups should get stored uncompressed since they aren't very large (compared to offline maps etc.).
Or OsmAnd should have an internal mechanism to restore favourites from one of the backups.

**Comment by vshcherb** [comment]
You can do it only on PC (unpack bz2), it should be a rare occasion these backups are needed.

**Comment by pebogufi** [comment]
https://play.google.com/store/apps/details?id=com.ghisler.android.TotalCommander
can do it, and not only that...

**Reply by anten1222** [reply]
I don't have a pc.  If you think the backup is not needed, it is not necessary to create it, I accidentally deleted my favorites and I need it very much

**Reply by pebogufi** [reply]
What is now your problem ? 
There was same more hint for you (Total Cmd). Did you read that, do you understand it ?

**Comment by anten1222** [comment]
ok imagine there are people who use WhatsApp and 2 other programs, to whom such a suggestion borders on ingnoratz.  If I don't want to understand a serious problem, I don't understand either.  Furthermore, I have often recommended Osmand very many people give up after a while, too complicated.--Diese Nachricht wurde von meinem Android Mobiltelefon mit GMX Mail gesendet.Am 28.07.21, 09:11 schrieb pebogufi ***@***.***>:

   
  What is now your problem ? There was same more hint for you (Total Cmd). Did you read that, do you understand it ?

**Reply by anten1222** [reply]
ok imagine there are people who use WhatsApp and 2 other programs, to whom such a suggestion borders on ingnoratz.  If I don't want to understand a serious problem, I don't understand either.  Furthermore, I have often recommended Osmand very many people give up after a while, too complicated.

**Reply by anten1222** [reply]
I use Googel for the translation.  I hope that I will be understood anyway

**Reply by pebogufi** [reply]
???...

**Reply by anten1222** [reply]
What exactly did you not understand?
